ChangeLog

```diff
@@ -15,6 +15,8 @@ Release date: TBA
 * Fix issues with ``typing_extensions.TypeVar``.
 
 
+* Fix ``ClassDef.fromlino`` for PyPy 3.8 (v7.3.11) if class is wrapped by a decorator.
```

Comment:
I can move this to `2.14.0` if we want to skip `2.13.4`.

Reply:
Let's not skip, releasing is cheap (compared to fixing the milestones)